platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -189,25 +211,38 @@ define(
 
                 // Initialize positions
                 getSetPosition(getSize(
-                    $element.children().eq(anchor.reversed ? 2 : 0)[0]
+                  $element.children().eq(anchor.reversed ? 2 : 0)[0]
                 ));
 
                 // And poll for position changes enforced by styles
                 activeInterval = $interval(function () {
                     getSetPosition(getSetPosition());
                 }, POLLING_INTERVAL, 0, false);
-
                 // ...and stop polling when we're destroyed.
                 $scope.$on('$destroy', function () {
                     $interval.cancel(activeInterval);
                 });
 
+
                 // Interface exposed by controller, for mct-splitter to user
                 return {
-                    position: getSetPosition,
-                    toggleClass: toggleClass,
                     anchor: function () {
                         return anchor;
+                    },
+                    position: function (value) {
+                        if (arguments.length > 0) {
+                            setUserWidthPreference(value);
+                            return getSetPosition(value);
+                        } else {
+                            return getSetPosition();
+                        }
+                    },
+                    startResizing: function () {
+                        toggleClass('resizing');
+                    },
+                    endResizing: function (pos) {
```

Comment:
recognize this variable is named to avoid a conflict with `position` in outer scope.  Instead of abbreviating to prevent conflict, name it more specifically, i.e. `finalPosition`.